Formula/macvim.rb

```diff
@@ -45,6 +45,11 @@ def install
     # If building for 10.7 or up, make sure that CC is set to "clang".
     ENV.clang if MacOS.version >= :lion
 
+    # Needed to deal with unsupported -mno-fused-madd
+    if build.with?("python") && ["10.8.4", "10.8.5", "10.9.0", "10.9.1", "10.9.2"].include?(OS::Mac.full_version.to_s) && OS::Mac::Xcode.version.to_f == 5.1
+      ENV.append "ARCHFLAGS", "=-Wno-error=unused-command-line-argument-hard-error-in-future"
+    end
```

Comment:
Remind me where this flag comes from? Is it from python-config? Does it make sense to just regexp the python-config --cflags output?